lib/DocumentInspectorFactoryInterface.php

```diff
@@ -11,27 +11,29 @@
 
 namespace Sulu\Component\DocumentManager;
 
+use Sulu\Component\DocumentManager\DocumentManagerContext;
+
 /**
  * Document inspector factory.
  *
  * Allows the client to implement their own DocumentInspector implementations.
  *
- * As Document inspectors are dependent upon a document manager, and as the
- * number of document managers is unbounded it would be impractical to manually
- * set the document manager on each individual instance.
+ * As Document inspectors are dependent upon a document context, and as the
+ * number of document contexts is unbounded it would be impractical to manually
+ * set the document context on each individual instance.
  *
  * The factory should create the DocumentInspector instance using the given
- * document manager.
+ * document context.
  */
 interface DocumentInspectorFactoryInterface
 {
     /**
      * Create a new DocumentInspector instance using the given document
-     * manager.
+     * context.
      *
-     * @param DocumentManagerInterface
+     * @param DocumentManagerContext
      *
      * @return DocumentInspector
      */
-    public function getInspector(DocumentManagerInterface $manager);
+    public function getInspector(DocumentManagerContext $context);
```

Comment:
Yeah, and we still don't need more than a single instance right? I think YAGNI should apply here.